src/graphql/operations/votes.ts

```diff
@@ -47,6 +51,28 @@ export default async function(parent, args, context, info) {
     return Promise.reject('request failed');
   }
 
+  if (requestedFields.space && votes.length > 0) {
+    const spaceIds = votes.map(vote => vote.space.id);
+    const query = `
```

Comment:
Have we thought about using knex for building SQL query? Its a bit more declarative. Though not requires. since we don't do too many db queries anyway.